src/main/java/ma/hibernate/util/HibernateUtil.java

```diff
@@ -0,0 +1,19 @@
+package ma.hibernate.util;
+
+import org.hibernate.SessionFactory;
+import org.hibernate.cfg.Configuration;
+
+public class HibernateUtil {
```

Comment:
Where we use this class?

Reply:
Without this class Trevis do not pass checks.  It is could be removed, but need to delete unused import at test, and I am not sure is it ok to fix tests )